src/SourceBuild/tarball/content/tools-local/init-build.proj

```diff
@@ -166,7 +159,7 @@
   </Target>
 
   <Target Name="PoisonPrebuiltPackages"
```

Comment:
Logged https://github.com/dotnet/source-build/issues/2317 for investigation of how poisoning should work now.